Source/Elasticurl/main.swift

```diff
@@ -238,20 +261,32 @@ struct Elasticurl {
             let semaphore = DispatchSemaphore(value: 0)
 
             var stream: HttpStream?
-            var connection: HttpClientConnection?
 
             let httpRequest: HttpRequest = HttpRequest(allocator: allocator)
-            httpRequest.method = "GET"
-            httpRequest.path = "/"
+            httpRequest.method = context.verb
+            let path = context.url.path == "" ? "/" : context.url.path
```

Comment:
Is this still necessary now that you're using a URL?

Reply:
hm good question, let me test without it really fast

Reply:
answer is yes, still comes in as an empty string even with `URL` object in swift unless the url itself had a slash at the end.